### ISSUE TITLE
Remove "?" from filenames instead of replacing it with "_"

### ISSUE THREAD

**Issue by huserbg** [gchudov/cuetools.net#108]
It's good that CUETools replaces the special characters with underscores but there is one specific case which I would like to be handled differently - track titles which end with a question mark.
I think that it's ugly to have filenames ending with "_" and it's not rare to have titles which are a question and end with "?".
What I do is manually rename the file to remove the "_" and edit the .cue file, but it will be great if this can be handled by CUETools itself.
It can be done more universal by allowing to specify a list of special characters to be removed instead of be replaced with "_".

**Comment by bvanevery** [comment]
Non-uniform processing of characters according to special cases sounds like a bad idea.  What if a title has multiple question marks in a row?  What if a title is 2 phrases, each ending with a question mark?  It is least surprising to handle all cases in a uniform manner.  Suggest closing this feature request.
